pkg/subctl/cmd/diagnose/cni.go

```diff
@@ -82,7 +82,12 @@ func checkCNIConfig(cluster *cmd.Cluster) bool {
 		return false
 	}
 
-	status.EndWithSuccess("The detected CNI network plugin (%q) is supported", cluster.Submariner.Status.NetworkPlugin)
+	if cluster.Submariner.Status.NetworkPlugin == constants.NetworkPluginGeneric {
```

Comment:
I think in case of Generic network plugin, the code will return false on L82 and will not reach this line

Reply:
Oops, I accidentally removed the Generic from the list of supported plugins. Good catch @maayanf24